run-qa-checks

```diff
@@ -6,7 +6,8 @@ echo ''
 echo 'Checking documentation build status'
 ./runsphinx-build
 
-if [[ $TRAVIS_PYTHON_VERSION != 3.6 ]]; then
+PYTHON_VERSION=$(python -c 'import sys; print(".".join(map(str, sys.version_info[:2])))')
+if [[ $PYTHON_VERSION != 3.6 ]]; then
```

Comment:
This code makes it independent of the platform it runs on. Sure, it does not look pretty but, it gets the job done.

Reply:
Is great, thanks